### DISCUSSION TITLE
Move save-bookmark icon from menu to out and make it explicitly visible

### DISCUSSION THREAD

**Discussion by krundru** [dailydotdev/daily#699] [Feature Requests]
Can we move the save-bookmark icon to the article's icon area next to upvote or comment

**Comment by idoshamun** [comment]
Hi,

Thanks for the suggestion this is a dup of #638.
Let's keep the discussion there.

**Comment by nimrodkra** [comment]
@krundru quick update: the bookmark button is back to the card. Thanks for the feedback 

![IMG_8444](https://github.com/dailydotdev/daily/assets/41470823/79559632-9b96-4302-a7ce-444b8718b8ed)
